### PR TITLE
onChangeTrackedCodes available for android.

### PR DESCRIPTION
**Pull request by TomekJurkowski**

### CODE REVIEW

File: docs/Scandit.java [hunk 1]
```diff
@@ -100,6 +100,11 @@
     */
     public Quadrilateral convertedPredictedLocation;
 
+    /**
+    * @brief The converted location of the barcode in the coordinate system of the picker. (Only available for tracked codes if MatrixScan is enabled.)
+    */
+    public Quadrilateral convertedLocation;
```
Comment (TomekJurkowski) [inline docs/Scandit.java]: @moritzhartmeier don't we also want to pass `location` ? we have `predictedLocation` `convertedPredictedLocation` and `convertedLocation` right now

Comment (petradonka) [reply docs/Scandit.java]: i think we should be passing location, did i forget to add it to the docs? 🤔

File: android/src/main/java/com/scandit/reactnative/BridgeHelpers.kt [hunk 2]
```diff
@@ -36,18 +37,44 @@ fun sessionToMap(scanSession: ScanSession?): WritableMap {
...
+    map.putArray("newlyTrackedCodes", trackedBarcodesToArray(newlyTrackedCodes, picker))
+
+    return map
+}
+
+private fun trackedBarcodesToArray(codes: Map<Long, TrackedBarcode>, picker: BarcodePicker): WritableArray {
+    val array = Arguments.createArray()
+
+    codes.forEach { _, barcode ->
+        val codeMap = Arguments.createMap()
```
Comment (petradonka) [inline android/src/main/java/com/scandit/reactnative/BridgeHelpers.kt]: do i get it right that you start from an empty dictionary? so the properties below are not in addition to the normal barcode properties but they're the only ones?

File: docs/Scandit.java (hunk 1 above)
Comment (petradonka) [reply docs/Scandit.java]: i just checked and it's there, a bit above and it actually says that it's only available for matrix scan tracked codes, which is not true, would you mind removing that sentence?

Comment (TomekJurkowski) [reply docs/Scandit.java]: hmm, but we are not really adding this property to the matrixscansession json, are we?

File: android/src/main/java/com/scandit/reactnative/BridgeHelpers.kt (hunk 2 above)
Comment (TomekJurkowski) [reply android/src/main/java/com/scandit/reactnative/BridgeHelpers.kt]: yea... that's why I might be missing the location then, right?

File: docs/Scandit.java (hunk 1 above)
Comment (petradonka) [reply docs/Scandit.java]: all barcodes (including tracked codes) should have a location property 🤔

File: android/src/main/java/com/scandit/reactnative/BridgeHelpers.kt (hunk 2 above)
Comment (petradonka) [reply android/src/main/java/com/scandit/reactnative/BridgeHelpers.kt]: yes, seems like it 😄

Comment (TomekJurkowski) [reply android/src/main/java/com/scandit/reactnative/BridgeHelpers.kt]: will fix it then 👍

Comment (TomekJurkowski) [reply android/src/main/java/com/scandit/reactnative/BridgeHelpers.kt]: Done

File: docs/Scandit.java (hunk 1 above)
Comment (TomekJurkowski) [reply docs/Scandit.java]: Done

**Pull request merged**